lib/fortnox/api/types/enums.rb

```diff
@@ -21,6 +21,12 @@ def self.default
         'COOKING','SNOWPLOWING','GARDENING','BABYSITTING','OTHERCARE',
         'TUTORING','OTHERCOSTS'
       )
+      CurrentHouseWorkTypes = Types::Strict::String.enum(
+        'CONSTRUCTION','ELECTRICITY','GLASSMETALWORK','GROUNDDRAINAGEWORK',
+        'MASONRY','PAINTINGWALLPAPERING','HVAC','CLEANING','TEXTILECLOTHING',
+        'SNOWPLOWING','GARDENING','BABYSITTING','OTHERCARE', 'OTHERCOSTS'
+      )
+      LegacyHouseWorkTypes = Types::Strict::String.enum( 'COOKING', 'TUTORING' )
```

Comment:
No good. Now you are duplicating the information in the types. The full enum, `HouseWorkTypes`, should consist of the other two, not duplicate all the values. That is why I suggested constants for the values, two constant arrays, that are concatenated to give the type.